### PR TITLE
moving hardcoded values to configuration file

### PR DESCRIPTION
**Pull request by jeffreyscarpenter**

### CODE REVIEW

File: killrvideo/__init__.py [hunk 1]
```diff
@@ -49,8 +53,10 @@ def serve():
             time.sleep(10)
 
     # Initialize Cassandra Driver and Mapper
-    profile = ExecutionProfile(consistency_level = ConsistencyLevel.LOCAL_QUORUM)
-    cluster = Cluster(contact_points=['10.0.75.1'], execution_profiles={EXEC_PROFILE_DEFAULT: profile})
+    profile = ExecutionProfile(consistency_level =
+                               ConsistencyLevel.name_to_value[default_consistency_level])
```
Comment (amandamoran) [inline killrvideo/__init__.py:57]: Just to make sure I understand. In the config file you would enter in a string value for the consistency level?

File: killrvideo/config.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "SERVICE_PORT": 8899,
+  "SERVICE_HOST": "10.0.75.1",
+  "ETCD_PORT": 2379,
+  "CONTACT_POINTS": ["10.0.75.1"],
+  "DEFAULT_CONSISTENCY_LEVEL": "LOCAL_QUORUM"
```
Comment (amandamoran) [inline killrvideo/config.json:6]: Question answered here :)

**Review by amandamoran — APPROVED** [review]
Looks good!

File: killrvideo/__init__.py (hunk 1 above)
Comment (jeffreyscarpenter) [reply killrvideo/__init__.py:57]: yes, text value

**Pull request merged**